gokart/file_processor.py

```diff
@@ -2,6 +2,7 @@
 import os
 from abc import abstractmethod
 from logging import getLogger
+import xml.etree.ElementTree as ET
```

Comment:
[IMO] for unity of the codes,  I recommend `et` instead of `ET` (for example, pandas as pd)

Reply:
In this case, I followed [Official docs](https://docs.python.org/ja/3/library/xml.etree.elementtree.html). Thanks anyway!